### PR TITLE
Instrument popup width fix

### PR DESCRIPTION
**Pull request by FBRTMaka**
@dlangston , @DanielJMaher , Hey DanL, fixed a couple things (error on an import and the sorting was broke).  Please review this morning.

@DanielJMaher , this also solves the problem of not having streams when selecting from the popup,

### CODE REVIEW

**Comment by FBRTMaka** [comment]
@birdage , since langston isn't going to be available this morning would you mind reviewing this?

**Comment by dlangston** [comment]
@FBRTMaka looking at it now

File: ooiui/static/css/common/scienceLayout.css [hunk 1]
```diff
@@ -503,13 +510,87 @@ img#arrayImg {
...
+  overflow-x: hidden;
+  overflow-y: auto;
+}
+
+
+#popupInstrumentTable.nasdaq tbody::-webkit-scrollbar {
+  width: 1px;
+  background: transparent;
+  border: 1px solid #D1DAF5;
+}
```
Comment (dlangston) [inline ooiui/static/css/common/scienceLayout.css]: A lot of these can be removed sense we aren't using the sticky-headers.

File: ooiui/static/js/views/common/MapView.js [hunk 2]
```diff
@@ -244,6 +244,8 @@ var MapView = Backbone.View.extend({
             }
         }
         $('#popupInstrumentTable').tablesorter({ sortList: [[0,0]]});
+                $("#popupInstrumentTable").stickyTableHeaders();
```
Comment (dlangston) [inline ooiui/static/js/views/common/MapView.js]: This can be removed

File: ooiui/static/css/common/scienceLayout.css (hunk 1 above)
Comment (FBRTMaka) [inline ooiui/static/css/common/scienceLayout.css]: @dlangston which ones?

**Comment by FBRTMaka** [comment]
@dlangston , ok, removed that. Thanks! - please merge.

File: ooiui/static/css/common/scienceLayout.css (hunk 1 above)
Comment (dlangston) [inline ooiui/static/css/common/scienceLayout.css]: these for sure

+#assembly-scroll-block.scrollable-block table {
+  margin-top: 2px;
+}
+
+#assembly-scroll-block.scrollable-block {
+  padding: 70px 0px;
+  margin-top: -72px;
+  margin-bottom: -72px;
+  margin-left: 0px;
+}
+
+#assembly-scroll-area.scrollable-area {
+  width: 100%;
+  max-height: 305px;
+  border: 2px solid #fff;
+  overflow-x: hidden;
+  overflow-y: auto;
+}

**Comment by FBRTMaka** [comment]
@dlangston and, in the future dan try a shorter name for your branches :D

File: ooiui/static/css/common/plotting.css [hunk 3]
```diff
@@ -289,7 +289,11 @@ div.bootstrap-datetimepicker-widget.dropdown-menu.bottom {
   width: auto;
   white-space: inherit;
 }
-
+.nasdaq tbody::-webkit-scrollbar {
+  width: 1px;
+  background: transparent;
+  border: 1px solid #D1DAF5;
+}
```
Comment (dlangston) [inline ooiui/static/css/common/plotting.css]: @FBRTMaka  this can also be removed, it is duplicate code.

**Comment by FBRTMaka** [comment]
@DanielJMaher , @birdage This is fine, can someone please merge this?

**Comment by FBRTMaka** [comment]
@dlangston , I'm not fixing all your redundancies...just what was failing.  Fix your own shit! (<3 U) :D

**Comment by dlangston** [comment]
fine...i will fix my own shit! lol

**Comment by dlangston** [comment]
@FBRTMaka should I merge this?

**Comment by FBRTMaka** [comment]
Yes please

**Comment by FBRTMaka** [comment]
And by yes please I mean merge it right now

**Pull request merged**